src/api/concero/socialNetworks/disconnectNetwork.ts

```diff
@@ -3,11 +3,10 @@ import { post } from '../../client'
 import { type Address } from 'viem'
 
 export const disconnectNetwork = async (address: Address, network: keyof IUser['subscriptions']): Promise<boolean> => {
-	const url = `${process.env.CONCERO_API_URL}/users/disconnectNetwork`
+	const url = `${process.env.CONCERO_API_URL}/disconnect/${network}`
```

Comment:
why is the endoint named disconnect? this is not obvious

Reply:
i fixed it